components/Plants/ShowAlPlantsThisPlaceList/index.js

```diff
@@ -0,0 +1,49 @@
+import { useRouter } from "next/router";
```

Comment:
You have so many PlantList component that it is quite hard to understand the purpose of each purpose. Maybe you can refactor this pat to have only one or two PlantList components. (Something for a refactoring PR :))

Reply:
You're absolutely right. Something like this happens to me when I have an idea but still have to try too much. More planning and sticking to the plan will do me and the app good.